packages/visualizations/src/components/Chart/Chart.svelte

```diff
@@ -112,8 +112,8 @@
                         // If the value is a percentage, we need to format the raw value
                         const formattedRawValue =
                             percentaged &&
-                            raw !== undefined &&
                             typeof raw === 'number' &&
+                            !Number.isNaN(raw) &&
```

Comment:
I might be missing something, but don't we want to return `(0)` if the raw value is `null` or `undefined`? If so, the current code won't returns the right value, it will return `false` if `raw === null` or `raw === undefined`.  

`Number.isNaN` doesn't have the same behaviour as `isNaN`: https://stackoverflow.com/a/33164924

Reply:
Indeed we do want it to return `(0)` but that's what the last `(${format(raw)})` does
However, the eslint rule does not allow the use of `isNaN` but only the use of `Number.isNaN` which is quite good because we already check that `raw` is a number, we should not check it again with `isNaN`. We just need to cross the `NaN` value as it is of type `number` too

Reply:
Oh right I just got your point! That's a good question, TBH I don't know what is best : display null to show that there is an error, or display 0

Reply:
Sorry I didn't know about the eslint rule, `Number.isNaN` is good enough you're right.

A little bit of code my worth a thousand words ^^ Here a little codepen illustrating the issue with the current code: https://codepen.io/wmai/pen/yLQZryL

What do you think?

Reply:
I'd say let's keep it simple, let's display `(0)`.